test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -777,6 +808,10 @@ public int nodeAndClientId() {
             return nodeAndClientId;
         }
 
+        public boolean isMasterNode() {
```

Comment:
I know I know. I just thought that it would add clarity though, cause is MasterNode is really ambiguous. I don't think it breaks any convention if we use the right terminology when adding new methods, especially in test code.